test/config_parser_SUITE.erl

```diff
@@ -266,7 +274,26 @@ hosts(_Config) ->
     ?err(parse(#{<<"general">> => #{<<"hosts">> => [<<>>]}})),
     ?err(parse(#{<<"general">> => #{<<"hosts">> => []}})),
     ?err(parse(#{<<"general">> => #{<<"hosts">> => [<<"host1">>, <<"host1">>]}})),
-    ?err(mongoose_config_parser_toml:parse(#{<<"general">> => #{}})). % hosts are mandatory
+    % either hosts or host_types must be provided
+    ?err(mongoose_config_parser_toml:parse(#{<<"general">> => #{}})).
+
+host_types(_Config) ->
+    %% ./rebar3 ct --suite config_parser_SUITE --group general --case host_types
+    ?eq([#config{key = host_types, value = [<<"type1">>, <<"type2">>]}],
```

Comment:
There is a test case that would fail in the current code:

`parse(#{<<"general">> => #{<<"host_types">> => [<<"my host type">>], <<"hosts">> => []}})`

Could you add it?

Reply:
as discussed in other thread, this won't fail any more.